src/LaunchDarkly.Client/EventProcessor.cs

```diff
@@ -64,35 +62,24 @@ public void Flush()
 
         private void BulkSubmit(IEnumerable<Event> events)
         {
+            var uri = new Uri(_config.BaseUri + "api/events/bulk");
```

Comment:
This isn't the correct baseUri-- you want the events BaseURI. But when you change that this is going to be the wrong path.